tests/analyze.R

```diff
@@ -59,12 +59,12 @@ Log <- function(...) {
   cat('\n')
 }
 
-LoadInputs <- function(prefix, ctx) {
+LoadInputs <- function(prefix_params, prefix_counts, ctx) {
   # prefix: path prefix, e.g. '_tmp/exp'
-  p <- paste0(prefix, '_params.csv')
-  c <- paste0(prefix, '_counts.csv')
-  m <- paste0(prefix, '_map.csv')
-  h <- paste0(prefix, '_hist.csv')
+  p <- paste0(prefix_params, '_params.csv')
```

Comment:
Rename these vars to indicate that some things are per test case and some are per test case run.